frontend/web/src/features/hostdashboard/hostcalen/components/Toolbar.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from "react";
+import { formatYearMonth } from "../utils/date";
+import { Auth } from "aws-amplify";
+import { getAccessToken } from "../utils/getAccessToken";
+
+export default function Toolbar({ view, setView, cursor, onPrev, onNext }) {
+  const [accommodations, setAccommodations] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [userId, setUserId] = useState(null);
+
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const user = await Auth.currentAuthenticatedUser();
+        const hostId =
+          user?.attributes?.["custom:hostId"] || user?.attributes?.sub || null;
+        setUserId(hostId);
+      } catch (err) {
+        console.error("Auth error fetching user id:", err);
+      }
+    })();
+  }, []);
+
+  useEffect(() => {
+    if (!userId) return;
+
+    const fetchAccommodations = async () => {
+      setIsLoading(true);
+      try {
+        const url = new URL(
+          "https://wkmwpwurbc.execute-api.eu-north-1.amazonaws.com/default/property/bookingEngine/byHostId"
+        );
+        url.searchParams.set("hostId", userId); 
+
+        const token = getAccessToken();
+        const res = await fetch(url.toString(), {
+          method: "GET",
+          headers: {
+            Authorization: token ?? "",
```

Comment:
If you send a empty string to the backend, you will get a 404 back. Maybe throw a error when the token is empty instead of sending a empty string, because a empty string will just give you a 404 back and not tell anything informative to the user. or just probably don't send a empty string at all

```suggestion
            Authorization: token,
```